ix-dev/community/gramps-web/ix_values.yaml

```
@@ -0,0 +1,14 @@
+images:
+  image:
+    repository: ghcr.io/gramps-project/grampsweb
+    tag: v25.5.1
+  redis_image:
+    repository: bitnami/redis
+    tag: 8.0.0
+
+consts:
+  gramps-web_container_name: gramps-web
+  gramps-web_celery_container_name: gramps-web_celery
```

```suggestion
  web_container_name: web
  celery_container_name: celery
```

Adjust labels as well and inside the compose